tools/data/diving48/README_zh-CN.md

```diff
@@ -0,0 +1,129 @@
+# 准备 Diving48
+
+## 简介
+
+<!-- [DATASET] -->
+
+```BibTeX
+@inproceedings{li2018resound,
+  title={Resound: Towards action recognition without representation bias},
+  author={Li, Yingwei and Li, Yi and Vasconcelos, Nuno},
+  booktitle={Proceedings of the European Conference on Computer Vision (ECCV)},
+  pages={513--528},
+  year={2018}
+}
+```
+
+用户可参考该数据集的 [官网](http://www.svcl.ucsd.edu/projects/resound/dataset.html)，以获取数据集相关的基本信息。
+在数据集准备前，请确保命令行当前路径为 `$MMACTION2/tools/data/diving48/`。
+
+## 步骤 1. 下载标注文件
+
+用户可以使用以下命令下载标注文件。
+
+```shell
+bash download_annotations.sh
+```
+
+## 步骤 2. 准备视频
+
+用户可以使用以下命令下载视频。
+
+```shell
+bash download_videos.sh
+```
+
+## Step 3. 抽取 RGB 帧和光流
+
+如果用户只想使用视频加载训练，则该部分是 **可选项**。
+
+用户可以使用以下命令从数据集 [官网](http://www.svcl.ucsd.edu/projects/resound/dataset.html) 下载 RGB 和光流的压缩包。
+
+```shell
+bash download_frames.sh
+```
+
+但是，这里提供的压缩包并不完整。若想获取完整的数据，可以使用以下步骤解帧。
```

Comment:
If the frames can not be used, what about removing that part (which might be a little misleading)?